crosstool/osx_cc_configure.bzl

```diff
@@ -156,9 +156,13 @@ def configure_osx_toolchain(repository_ctx):
         Label("@build_bazel_apple_support//crosstool:wrapped_clang.cc"),
     ))
 
-    (xcode_toolchains, xcodeloc_err) = run_xcode_locator(repository_ctx, xcode_locator)
-    if not xcode_toolchains:
-        return False
+    skip_xcode_fetch = "BAZEL_SKIP_XCODE_FETCH" in repository_ctx.os.environ and repository_ctx.os.environ["BAZEL_SKIP_XCODE_FETCH"] == "1"
```

Comment:
What do you think about using [bool](https://bazel.build/rules/lib/globals#bool) to evaluate the environment variable value? It allows folks to pick their favorite representation.

Reply:
this is what the upstream CC toolchain vars enforce so I figure we might as well keep the same behavior